### ISSUE TITLE
ArcadeDB Benchmarks

### ISSUE THREAD

**Issue by C-monC** [ArcadeData/arcadedb#410]
Hi,

I saw in issue #384 you're working on benchmarks.
Is it possible for you to post your preliminary findings and update as you go?

I can assist with the studies as well.

**Comment by lvca** [comment]
@C-monC Are you interested on having a benchmark with other multi-model or graphdb DBMSs? Or maybe implementing an existent benchmark?

**Comment by Extarys** [comment]
I would be curious to compare Postgres/Yugabyte and maybe AGE or JanusGraph :)
Found out this project today, gonna spin a container and give it a go

**Comment by lvca** [comment]
@Extarys a graph benchmark?

**Comment by Extarys** [comment]
Doesn't it support graph? Oops, I read it was supported on the main page. Maybe I have no idea what I'm talking about :/

**Comment by lvca** [comment]
Yep. Graphs are native in ArcadeDB, but I don't think are in Yuga.

**Comment by gramian** [comment]
https://db-benchmarks.com is a set of relational benchmarks, but maybe nonetheless interesting.

**Comment by lvca** [comment]
Implemented LDBC benchmark: https://arcadedb.com/blog/graph-olap-engine-the-fastest-graph-analytics-with-zero-compromises/
